internal/cli/iam/organizations/apikeys/accesslists/create.go

```diff
@@ -107,13 +105,6 @@ func CreateBuilder() *cobra.Command {
 			)
 		},
 		RunE: func(_ *cobra.Command, _ []string) error {
-			if opts.currentIP {
-				publicIP := store.IPAddress()
```

Comment:
any reason to drop this option?

Reply:
Because this never worked for MongoCLI, I can make it work, but it never did: https://github.com/mongodb/mongodb-atlas-cli/blob/20ee56764799694003370b88cf81a4312236cc6e/internal/store/ip_info.go#L31

Reply:
cool if it never worked then ok to delete, we need to document for the release notes all removals